packages/server/tests/acceptance/hbarLimiter.spec.ts

```diff
@@ -232,12 +232,12 @@ describe('@hbarlimiter HBAR Limiter Acceptance Tests', function () {
         }
       });
 
-      describe('Total HBAR Limit', () => {
+      describe('@hbarlimiter-batch1 Total HBAR Limit', () => {
         it('should execute "eth_sendRawTransaction" without triggering HBAR rate limit exceeded', async function () {
           const parentContract = await deployContract(parentContractJson, accounts[0].wallet);
 
           // awaiting for HBAR limiter to finish updating expenses in the background
-          await Utils.wait(6000);
+          await Utils.wait(10000);
```

Comment:
Maybe not an issue to address in this PR but waits are a code smell in tests and needing to increase them confirms there's a coordination logic not being properly incorporated into the tests.
Can we open a ticket to address this in the future. What are we actually waiting on and can that status be polled for instead of a generic wait?